src/commands/aem-migration/all.js

```diff
@@ -72,6 +83,54 @@ async function runRepositoryModernizer(config, command) {
     }
 }
 
+async function runWorkflowMigrator(config, command) {
+    helper.clearOutputFolder(constants.TARGET_WORKFLOW_FOLDER);
+    command.log("\n********** Executing Workflow Migrator **********");
+    if (!helper.isWorkflowConfigValid(config.workflowMigrator)) {
+        // invalid wf-migrator related configs found.
+        command.log(`Invalid configuration! Please check ${Commons.constants.LOG_FILE} for more information.\n`);
+        return;    
+    }
+    helper
+        .fetchLatestReleasedAsset(
+            constants.WF_MIGRATOR_REPO_OWNER,
+            constants.WF_MIGRATOR_REPO_NAME,
+            constants.WF_MIGRATOR_JAR
+        )
+        .then(async () => {
+            Commons.logger.info(
+                `Downloading of ${constants.WF_MIGRATOR_JAR} from ${constants.WF_MIGRATOR_REPO_NAME} successful.`
+            );
+            for (const project of config.workflowMigrator.projects) {
+                if (
```

Comment:
@dichaudhary I think we should also run the WF migrator on the specified paths, if the corresponding target project path doesn't resolve. Ideally logic should be to run on target project path IF present, ELSE run on the path mentioned in the config.